anthos-bm-edge-deployment/.gitignore

```diff
@@ -0,0 +1,99 @@
+target/
+pkg/
+tmp/
+
+.gitcookies
+
+.screenrc
+
+# Newrelic agent log
+newrelic_agent.log
+
+# test
```

Comment:
Question: 
Was this `# test` comment intentional?

Reply:
Removed in https://github.com/GoogleCloudPlatform/anthos-samples/pull/377/commits/dd063e8dd48dcd0ba2c4de190174b26253bc1b38